### PR TITLE
[new] componentjs

### PR DESCRIPTION
**Pull request by AuspeXeu**
Added componentjs library

### CODE REVIEW

File: ajax/libs/componentjs/package.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
+{
+    "name":        "componentjs",
+    "npmname":     "componentjs",
+    "npmFileMap": [
+        {
+          "basePath": "/bld/",
```
Comment (PeterDaveHello) [inline ajax/libs/componentjs/package.json]: I can't find this path under its folder

**Comment by AuspeXeu** [comment]
Removed that part.

**Comment by PeterDaveHello** [comment]
But we still need auto-update config, is it possible? thanks!

**Comment by AuspeXeu** [comment]
That should do, right? It's a different build repository I added it under the repositories property but left the repository property as is to point to the source repo.

**Comment by PeterDaveHello** [comment]
Thanks, looks okay, would you please squash these commits into only 1 commit as our document said? Thank you!

**Pull request closed without merging**